src/linker/utilities/general_utils.py

```diff
@@ -103,3 +103,7 @@ def exit_with_validation_error(error_msg: Dict) -> None:
         "\n==========================================\n"
     )
     exit(errno.EINVAL)
+
+
+def merge_dicts(dicts: List[Dict]):
+    return {key: val for dict in dicts for key, val in dict.items()}
```

Comment:
Is there no method in `collections` that does this?

Reply:
I tried looking through some standard libraries, but i didn't understand how ChainMap worked. This [PEP showed me how](https://peps.python.org/pep-0584/)
so i think `dict(ChainMap(*dicts))` wins